perllib/FixMyStreet/Cobrand/Buckinghamshire.pm

```diff
@@ -126,6 +126,30 @@ sub open311_contact_meta_override {
     } if $service->{service_name} eq 'Flytipping';
 }
 
+sub process_open311_extras {
+    my ($self, $c, $body, $extra) = @_;
+
+    my $report = $c->stash->{report};
+    if ($report->bodies_str =~ /,/) {
```

Comment:
Is this relying on only the Flytipping category begin sent to both District and County so it'll be the only one with a comma in the `bodies_str`? That sort of feels like we should have a `return unless $report->category eq "Flytipping"`; check first.

Reply:
Looks like Trees are also sent to both, so yes, thanks!